packages/node_modules/@ciscospark/react-component-incoming-call/src/index.js

```diff
@@ -53,11 +53,12 @@ function IncomingCall({
 
   return (
     <div className={classNames(styles.callInactiveContainer, 'call-inactive-container')}>
-      <div className={classNames(styles.callAvatar, 'call-avatar')}>
```

Comment:
Can we remove the `callAvatar` style from the css file since it is no longer used?